{{ cookiecutter.project_slug }}/compose/start-devserver.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+#
+# Run a database migration and start the development server.
+set -xe
+
+cd /usr/src/app
+
+python ./manage.py migrate
+
+exec python ./manage.py runserver 0.0.0.0:8080
```

Comment:
can we use the default django port (8000) for runserver to avoid confusion?

Reply:
Wilko. Do we want to fix up the other projects which use 8080? (E.g. [lookupproxy uses 8080](https://github.com/uisautomation/lookupproxy/blob/master/docker-compose.yml))